### DISCUSSION TITLE
Writing to Custom UDT

### DISCUSSION THREAD

**Discussion by chrisegge** [dmroeder/pylogix#204] [Q&A] [answered]
I am trying to write to a custom UDT - STRING5 - A string limited to 5 characters, I can write to it. But only if I write to the 5 SINT's first and then the LEN DINT. Any way around this?

**Comment by dmroeder** [comment, marked as answer]
That's all you can do currently.
https://github.com/dmroeder/pylogix/blob/master/examples/14_write_custom_string.py

**Reply by chrisegge** [reply]
That works for me! its a lot better than having an unreadable SINT value line by line haha
